yarn-project/circuits.js/src/scripts/constants.in.ts

```diff
@@ -84,9 +84,8 @@ function processConstantsTS(constants: { [key: string]: string }): string {
 function processConstantsCpp(constants: { [key: string]: string }): string {
   const code: string[] = [];
   Object.entries(constants).forEach(([key, value]) => {
-    // We exclude large numbers
-    if (CPP_CONSTANTS.includes(key) && !(value.startsWith('0x') || value.includes('0_0'))) {
-      code.push(`const size_t ${key} = ${value};`);
+    if (CPP_CONSTANTS.includes(key)) {
+      code.push(`#define ${key} ${value}`);
```

Comment:
I'm using defines so that we don't need to worry about the size here.

Reply:
But a big number will still need ll suffix no?

Reply:
IIUC it's not needed, but in any case really big constants will fail on use because c++ cannot hold anything > 64 bits natively. See ["The type of a literal"](https://en.cppreference.com/w/cpp/language/integer_literal).
> The type of the integer literal is the first type in which the value can fit, from the list of types which depends on which numeric base and which integer-suffix was used.